blogposts/announcing-sourcegraph-3.6.md

```diff
@@ -0,0 +1,162 @@
+---
+title: 'Sourcegraph 3.6: Bitbucket Server plugin, search performance improvements, and quick links'
+author: Christina Forney
+publishDate: 2019-07-20T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.6
+heroImage: /sourcegraph-mark.png
+published: true
+---
+
+[Sourcegraph](https://about.sourcegraph.com/) is the standard developer platform for code search and navigation at many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
```

Comment:
```suggestion
Sourcegraph is the standard developer platform for code search and navigation at many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
```

Reply:
This was intro in the previous releases, we don't want to include a link to the about site?

Reply:
Yeah, I'm not sure why I put it in here. If you want to keep it, just change the url to `/`.